src/main/java/org/c4sg/entity/Project.java

```diff
@@ -47,11 +46,11 @@
 	@Column(name = "zip")
 	private String zip;
 	
-	@Column(name = "remote_flag", columnDefinition="char(1)", nullable = false)
-	private Boolean remoteFlag;
+	@Column(name = "remote_flag", columnDefinition="char(1) default 'Y'", nullable = false)
+	private char remoteFlag='Y';
 
-    @Column(columnDefinition="char(1)", nullable = false)
-    private String status;
+    @Column(columnDefinition="char(1) default 'A'", nullable = false)
+    private String status="A";
     
 	@Column(name = "created_time", nullable = false)
 	@Value("${var.string:#{NULL}}")
```

Comment:
Please check issue # 75 fix #87. It's from that code change. Do you want to remove it from all 3 files: Project, Organization and User.java?